src/core/send_buffer.c

```diff
@@ -257,8 +257,8 @@ QuicSendBufferConnectionAdjust(
         return; // Nothing to do.
     }
 
-    const uint64_t NewIdealBytes =
-        QuicGetNextIdealBytes(Connection->CongestionControl.BytesInFlightMax);
+    const uint64_t NewIdealBytes = QuicGetNextIdealBytes(
+        QuicCongestionControlGetBytesInFlightMax(&Connection->CongestionControl));
```

Comment:
```suggestion
    const uint64_t NewIdealBytes =
        QuicGetNextIdealBytes(
            QuicCongestionControlGetBytesInFlightMax(&Connection->CongestionControl));
```
My preferring formatting.